<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayWindow</class>
 <widget class="QMainWindow" name="DisplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>508</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>508</width>
    <height>1000</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Demon</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/desir/Pictures/3418176f1061530c08a0a3f0d11cf870.ico</normaloff>C:/Users/desir/Pictures/3418176f1061530c08a0a3f0d11cf870.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;Unknown&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;Unknown&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="last_pushbutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Last</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_stop_pushbutton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_pushbutton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="war_situation">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>411</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="message_box">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
